packages/components/src/components/tabs/tabs.scss

```diff
@@ -43,6 +43,12 @@
         }
         &--header-fit-content {
             width: fit-content;
+
+            @include mobile {
+                display: flex;
+                justify-content: space-evenly;
+                width: 100vw;
+            }
```

Comment:
Probably should make this (`100vw`) opt-in through another prop if in the future some component doesn't want to have a full page width tabs component.

Reply:
Can you please explain how an `opt-in` works? I haven't come across it yet.